commit 01671608a3bddde369cdd42aed12e1c019b87158
Author: Siddhesh Poyarekar <siddhesh@sourceware.org>
Date:   Wed Aug 4 02:21:01 2021 +0530

    gethosts: Remove unused argument _type
    
    The generated code is unchanged.
    
    (cherry picked from commit b17e842a60819098d2a203ecc8b8371b7e1d6c65)

diff --git a/sysdeps/posix/getaddrinfo.c b/sysdeps/posix/getaddrinfo.c
index f391dc0a59849aab..702d8a50e0c218d2 100644
--- a/sysdeps/posix/getaddrinfo.c
+++ b/sysdeps/posix/getaddrinfo.c
@@ -239,7 +239,7 @@ convert_hostent_to_gaih_addrtuple (const struct addrinfo *req,
   return true;
 }
 
-#define gethosts(_family, _type) \
+#define gethosts(_family) \
  {									      \
   struct hostent th;							      \
   char *localcanon = NULL;						      \
@@ -864,7 +864,7 @@ gaih_inet (const char *name, const struct gaih_service *service,
 		      if (req->ai_family == AF_INET6
 			  || req->ai_family == AF_UNSPEC)
 			{
-			  gethosts (AF_INET6, struct in6_addr);
+			  gethosts (AF_INET6);
 			  no_inet6_data = no_data;
 			  inet6_status = status;
 			}
@@ -876,7 +876,7 @@ gaih_inet (const char *name, const struct gaih_service *service,
 				 know we are not going to need them.  */
 			      && ((req->ai_flags & AI_ALL) || !got_ipv6)))
 			{
-			  gethosts (AF_INET, struct in_addr);
+			  gethosts (AF_INET);
 
 			  if (req->ai_family == AF_INET)
 			    {
